components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1307,7 +1309,9 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
                                     return;
                                 }
                                 const userRepos = await services.repositoryProvider.getUserRepos(user);
-                                userRepos.forEach((r) => suggestions.push({ url: r.replace(/\.git$/, "") }));
+                                userRepos.forEach((r) =>
+                                    suggestions.push({ url: r.replace(/\.git$/, ""), priority: 5 }),
```

Comment:
Random side note: Would be cool to add `lastUse` here too (e.g. in order to first show repositories I've most recently contributed to, even if I've never opened them in Gitpod before).